README.md

```diff
@@ -131,94 +131,86 @@ WebAuthn.configure do |config|
 
-# For your frontend code, you might find @github/webauthn-json JS library useful.
-# Especially for handling the necessary decoding of the options.
+# For your frontend code, you might find @github/webauthn-json npm package useful.
+# Especially for handling the necessary decoding of the options, and sending the
+# `PublicKeyCredential` object back to the server.
 ```
 
 #### Verification phase
 
 ```ruby
-# These should be ruby `String`s encoded as binary data, e.g. `Encoding:ASCII-8BIT`.
-#
-# If the user-agent is a web browser, you would use some encoding algorithm to send what
-# `navigator.credentials.create` returned through the wire.
-#
-# Then you need to decode that data before passing it to the `#verify` method.
-#
-# E.g. in https://github.com/cedarcode/webauthn-rails-demo-app we use `Base64.urlsafe_decode64`
-# on the user-agent encoded data before calling `#verify`
-attestation_object = "..."
-client_data_json = "..."
-
-attestation_response = WebAuthn::AuthenticatorAttestationResponse.new(
-  attestation_object: attestation_object,
-  client_data_json: client_data_json
-)
-
+# Assuming you're using @github/webauthn-json package to send the `PublicKeyCredential` object back
+# in params[:publicKeyCredential]:
+webauthn_credential = WebAuthn::Credential.from_create(params[:publicKeyCredential])
 
 begin
-  attestation_response.verify(expected_challenge)
-
-  # 1. Register the new user and
-  # 2. Keep Credential ID, Credential Public Key and Sign Count under storage
-  #    for future authentications
-  #    Access by invoking:
-  #      `attestation_response.credential.id`
-  #      `attestation_response.credential.public_key`
-  #      `attestation_response.authenticator_data.sign_count`
-rescue WebAuthn::VerificationError => e
+  webauthn_credential.verify(session[:creation_challenge])
+
+  # Store Credential ID, Credential Public Key and Sign Count for future authentications
+  user.credentials.create!(
+    webauthn_id: webauthn_credential.id,
+    public_key: webauthn_credential.public_key,
+    sign_count: webauthn_credential.sign_count
+  )
+rescue WebAuthn::Error => e
   # Handle error
 end
 ```
 
-### Authentication
+### Credential Authentication
 
-#### Initiation phase
+> The ceremony where a user, and the user’s client (containing at least one authenticator) work in concert to cryptographically prove to a Relying Party that the user controls the credential private key associated with a previously-registered public key credential (see Registration). Note that this includes a test of user presence or user verification. [[source](https://www.w3.org/TR/webauthn-2/#authentication-ceremony)]
 
-Assuming you have the previously stored Credential ID, now in variable `credential_id`
+#### Initiation phase
 
 ```ruby
-get_options = WebAuthn::Credential.get_options(allow: credential_id)
+get_options = WebAuthn::Credential.get_options(allow: user.credentials.map { |c| c.webauthn_id })
```

Comment:
Same here about `.get_options`.

Would `.options_for_get` or `authentication_options` be less confusing?

Reply:
Fixed.